MakieCore/src/types.jl

```diff
@@ -68,6 +64,12 @@ end
 
 const px = Pixel(1)
 
+const Space = Type{<: Unit}
```

Comment:
This will be rough for some of your previous field types...if it's possible to standardize internally...

Reply:
Yeah, I think this was just a stop gap solution, to not trigger a bigger refactor... I think this can just be an enum going forward..